### PR TITLE
Add section on most common problems with output review requests

### PR DESCRIPTION
**Pull request by LFISHER7**
This adds a section to the output checking docs indicating the top 5 reasons for output review requests being rejected. This is based on a review of a sample of review requests performed in https://github.com/ebmdatalab/service-analytics-team/issues/148

### CODE REVIEW

File: docs/releasing-files.md [hunk 1]
```diff
@@ -287,7 +287,6 @@ Please run through this checklist before making a review request.
 4. Have you [redacted any low counts](#redacting-counts-less-than-or-equal-to-7)?
 5. Have you [rounded any counts](#rounding-counts) (including [counts underlying rates](#rounding-rates))?
 6. Have you supplied underlying counts for all of your results?
-7. Are all of the outputs clearly described?
```
Comment (Jongmassey) [inline docs/releasing-files.md:290]: Is this deletion intentional, and if so why?

File: docs/releasing-files.md [hunk 2]
```diff
@@ -318,6 +317,24 @@ Before any files are released from the secure server, they are checked independe
...
+    * Stating the incorrect file path. You should check all file paths point to the relevant files within your `release` folder before making a request.
+    * Files included in the review form being missing from the `review` folder.
+    * Using unclear column/variable names or poorly describing the presented data. See [here](#context-requirements) for more details on the context requirements.
+    * Not clearly indicating the relationship between different outputs.
+    * Where an output has previously been requests, not indicating how the output differs to previously reviewed version.
+3. **There are unredacted counts in the outputs**. Prior to rounding counts, [any counts <=7 should be redacted](#redacting-counts-less-than-or-equal-to-7). The redaction approach should be clearly described when making a review request. It is not uncommon for the stated redaction approach to be improperly implemented in the outputs. Inappropriate redaction of low counts accounts for **~20%** or rejected outputs.
```
Comment (Jongmassey) [inline docs/releasing-files.md:332]: s/or/of

File: docs/releasing-files.md [hunk 3]
```diff
@@ -318,6 +317,24 @@ Before any files are released from the secure server, they are checked independe
...
+4. **Underlying data is not provided**. To ensure the low number threshold is met, reviewers require to see the underlying data for each output. This includes the data used to generate figures and to calculate summary statistics such as mean or median. **~10%** or rejected outputs are due to underlying data not being provided.
```
Comment (Jongmassey) [inline docs/releasing-files.md:333]: s/or/of

File: docs/releasing-files.md (hunk 2 above)
Comment (Jongmassey) [reply docs/releasing-files.md:332]: "of rejected outputs"

File: docs/releasing-files.md (hunk 1 above)
Comment (LFISHER7) [reply docs/releasing-files.md]: This was accidental, i've reinstated it

**Comment by LFISHER7** [comment]
Thanks for spotting those! I've reinstated point 7 to the checklist and fixed the two typos

**Review by Jongmassey — APPROVED** [review]

**Pull request merged**